gradle/libs.versions.toml

```
@@ -77,6 +77,8 @@ androidx-test-truth = { module = "androidx.test.ext:truth", version = "1.6.0" }
 coil = { module = "io.coil-kt:coil", version.ref = "coil" }
 coilCompose = { module = "io.coil-kt:coil-compose", version.ref = "coil" }
 
+constraintsExplorer = { module = "com.zachklipp:constraints-explorer", version = "1.1.0" }
```

You don't need a complex toml object

```suggestion
constraintsExplorer = "com.zachklipp:constraints-explorer", version = "1.1.0"
```